### ISSUE TITLE
ECS agent tasks fail to start due to agent disconnection

### ISSUE THREAD

**Issue by CloudNiner** [azavea/temperate#1155]
tl;dr ECS agent was disconnected on an instance when I deployed this am, restarted docker and agent, we should probably update the ECS agent in our climate stacks.

#### Details

I deployed v1.5.0 this am, and when deploying, I ran into an issue where the new ECS tasks were failing to be picked up by valid container instances due to the error:

> The closest matching container-instance <container-instance-id> doesn't have the agent connected.

This led to a bit of googling, and found https://github.com/aws/amazon-ecs-agent/issues/517

So, I followed the steps taken by that issue, and I found this message in the ecs-agent log on instance i-007c52ecfe2dccf87:
```
2018-09-14T11:57:22Z [INFO] Managed task [arn:aws:ecs:us-east-1:784347171332:task/554ab54a-e5ed-4b14-bf6f-e2375a4dad48]: task at steady state: RUNNING
2018-09-14T11:57:33Z [WARN] Udev watcher reconciliation failed: udev watcher: unable to retrieve network interfaces: too many open files
```

When running a netstat on the number of open docker connections, I got:
```
[ec2-user@ip-10-0-3-38 ~]$ netstat -x | grep '/var/run/docker.sock' | wc -l
1981
```

This is way too high, and we're running a newer than the issue, but still outdated, version of the ECS agent. Perhaps it would be best to update to the latest ECS agent before digging further on this issue. Other googling seems to suggest that this issue continues to occasionally cause trouble.

In the meantime, to workaround the issue, I restarted the docker daemon with `sudo service docker stop && sudo service docker start` and then restarted the ecs agent with `sudo start ecs` according to the [AWS ECS agent troubleshooting guide](https://aws.amazon.com/premiumsupport/knowledge-center/ecs-agent-disconnected/). 

Attached is an ecs agent log from when the agent was disconnected, if that ends up being helpful.

[ecs-agent.log.2018-09-14-11-too-many-open-files.txt](https://github.com/azavea/temperate/files/2383360/ecs-agent.log.2018-09-14-11-too-many-open-files.txt)
